plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2167,8 +2171,15 @@ elementgui.living_entity.error_entity_needs_name=Entity needs a name
 elementgui.living_entity.page_visual_and_sound=Visual and sound
 elementgui.living_entity.page_behaviour=Behaviour
 elementgui.living_entity.page_particles=Particles
+elementgui.living_entity.entity_data_accessors= Entity data accessors
 elementgui.living_entity.page_ai_and_goals=AI and goals
 elementgui.living_entity.page_spawning=Spawning
+elementgui.living_entity.entity_data_entries=Entity data entries
+elementgui.living_entity.entity_data=<html>Entity data entries are values inside the entity and synchronised between the server and the client:<br>\
+  <small>It can be used to have different textures on a single entity.
```

Comment:
I don't see a reason to make the new page header with shorter version of the corresponding help tip used for the text.
I'd recommend using the border title text in its place and make the border untitled.